src/test/java/com/epam/tc/hw2/ex2/ExerciseSecondTest.java

```diff
@@ -0,0 +1,60 @@
+package com.epam.tc.hw2.ex2;
+
+import com.epam.tc.hw2.BrowserTestBase;
+import java.util.List;
+import java.util.Objects;
+import org.assertj.core.api.SoftAssertions;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.Select;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.annotations.Test;
+
+
+public class ExerciseSecondTest extends BrowserTestBase {
+
+    @Test
+    public void testDifferentElements() throws InterruptedException {
+        // 5. Open through the header menu Service -> Different Elements Page - Page is opened
+        driver.findElement(By.className("menu-title")).click();
+
+        new WebDriverWait(driver, 10)
+                .until(ExpectedConditions.elementToBeClickable(By.linkText("Different elements")));
+        driver.findElement(By.linkText("Different elements")).click();
+        SoftAssertions softAssertions = new SoftAssertions();
+        softAssertions.assertThat(driver.getTitle()).isEqualTo("Different Elements");
+
+        // 6. Select checkboxes (Water and Wind)
+        List<WebElement> checkBoxes = driver.findElements(By.className("label-checkbox"));
+
+        for (WebElement checkB : checkBoxes) {
+            if (Objects.equals(checkB.getText(), "Water") || Objects.equals(checkB.getText(), "Wind")) {
+                checkB.click();
+                softAssertions.assertThat(checkB.isEnabled()).isTrue();
+            }
+        }
```

Comment:
Please, try to find another solution... Rewrite your loop, or maybe it would be easier for you to have separate locators for checkboxes.